### ISSUE TITLE
Send bitcoin address via Airdrop

### ISSUE THREAD

**Issue by karek314** [voisine/breadwallet#372]
Would be great to add possibility to share address / payment request with airdrop.

**Comment by koalalorenzo** [comment]
I have to agree. It would be nice to share a tx via Airdrop in order to broadcast it when an internet connection is not available! (It happened to me once when I was abroad and without internet, I had to pay with credit card :-( )

**Comment by voisine** [comment]
@koalalorenzo, this issue is referring to sending an address or request via airdrop. Your case of sending the tx when the sender has no internet access is the more interesting one I think. Then Bitcoin payments can work so long as the recipient has access. Typically payment recipients are more likely to have it, as in the case of merchants with fixed shop locations. Unfotunatly it will require the merchant to support a new protocol. Rather than airdrop it would probably be better to use something that non-apple devices can also support like bluetooth.

**Comment by koalalorenzo** [comment]
Still better than nothing! I think that the qr-code scan it is much faster than AirDrop, but the "no-internet" feature should be considered!

**Comment by todb-r7** [comment]
I'd be worried about the security implications of implementing any Airdrop *or* Bluetooth functionality. With the existing QR Code scheme, you are merely relying on cameras and air. I cannot imagine many cases where you'd want to share an address in the normal local ranges where you also cannot just show your phone screen / camera with the other party.

**Comment by koalalorenzo** [comment]
True, but this sounds just like an excuse to not implement a feature. For example: lets not implement the touchscreen because we already can use the mouse everywhere. Do you see the point?

And that is where my point starts ( with is not included in this issue, but OT ) about the necessity to share the transaction too via AirDrop/BT. If you do that, you have another reason to implement this feature and have some more use cases for the app. (offline for example)
